site/styles/_meta-detail.scss

```diff
@@ -560,9 +568,15 @@
               display: none !important;
             }
           }
-
         }
       }
     }
   }
 }
+
+.meta-detail-bonus-content .synopsis {
+  font-size: 10px;
+  @include media-breakpoint-up(lg) {
+    font-size: 12px;
+  }
+}
```

Comment:
Also added this one, based on the changes in the Falcon repo.

Reply:
Let's ditch this as it could have been their specific styling.